highfidelity/hifi
# Adding Blocks Tablet App to Marketplace

**TEST PLAN**

1. Blocks app should load as default app (named "BLOCKS" with geometric shape icon)
2. Open "BLOCKS"
3. Click on any model
4. New page should open with model preview and "Download" button (if "Download" doesn't appear, that means the model is not remixable, so you'll have to right-click to go back to the item listings if you are in desktop mode or click the "Back" button if you are in HMD mode)
5. Click "Download"
6. Model should appear in front of you with colors
